Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/OnboardingSettings.kt

```diff
@@ -0,0 +1,38 @@
+package de.rki.coronawarnapp.storage
+
+import android.content.Context
+import androidx.core.content.edit
+import de.rki.coronawarnapp.util.di.AppContext
+import de.rki.coronawarnapp.util.preferences.clearAndNotify
+import org.joda.time.Instant
+import javax.inject.Inject
+
+class OnboardingSettings @Inject constructor(
```

Comment:
this class is injected in Application ,is not better to annotate it with Singleton?

Reply:
whoops, missed that, absolutely